website/project/model.py

```diff
@@ -75,16 +74,9 @@ def has_anonymous_link(node, auth):
     :param str link: any view-only link in the current url
     :return bool anonymous: Whether the node is anonymous to the user or not
     """
-    view_only_link = auth.private_key or request.args.get('view_only', '').strip('/')
```

Comment:
Five were added in 602f4738eac8c6f6ecd4927159cdccd04164542d, the parent of 87fa33c2309626dd99b2ac05ed20708ddb5bc7a4.